<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertPanel</class>
 <widget class="QWidget" name="ConvertPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>造形フォルダ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modelDirLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openFolderPathButton">
         <property name="text">
          <string>選択</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>層データ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="layerLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>現在は層ごとのみ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openLayerFolderButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>選択</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>開始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startLayerLineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>終了</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endLayerLineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>積層間隔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="layerPitchLineEdit">
         <property name="text">
          <string>4000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>変換後のファイルの名前</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="covertNameLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="convertPushButton">
       <property name="text">
        <string>変換</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
